src/scripts/party/evolutions/Evolution.ts

```diff
@@ -41,8 +42,22 @@ abstract class Evolution {
 
         App.game.party.gainPokemonById(PokemonHelper.getPokemonByName(evolvedPokemon).id, shiny, true);
 
-        // EVs
         const evolvedPartyPokemon = App.game.party.getPokemonByName(evolvedPokemon);
+        if (newPokemon && App.game.challenges.list.realEvolutions.active()) {
+            const basePartyPokemon = App.game.party.getPokemon(PokemonHelper.getPokemonByName(this.basePokemon).id);
+            evolvedPartyPokemon.effortPoints = basePartyPokemon.effortPoints;
+            evolvedPartyPokemon.pokerus = basePartyPokemon.pokerus;
+            evolvedPartyPokemon.shiny = evolvedPartyPokemon.shiny || basePartyPokemon.shiny;
+            evolvedPartyPokemon.attackBonusAmount = basePartyPokemon.attackBonusAmount;
+            evolvedPartyPokemon.attackBonusPercent = basePartyPokemon.attackBonusPercent;
+            evolvedPartyPokemon.proteinsUsed = basePartyPokemon.proteinsUsed;
+            App.game.party.removePokemonByName(this.basePokemon);
+            const basePokemonStatistics = App.game.statistics.pokemonCaptured[PokemonHelper.getPokemonByName(this.basePokemon).id];
+            App.game.statistics.pokemonCaptured[evolvedPartyPokemon.id](basePokemonStatistics());
+            basePokemonStatistics(0);
```

Comment:
Why do the capture statistics move? Also these don't move the gender specific statistics so you can have Total: 0, male: 1 on the basePokemon

Reply:
I really can't remember... There was something that used the statistics to see if the pokemon was caught, but that has been removed since, it looks like. At least i can't find it in the code.
I will revert it and hope for the best. should be easy to clean up, if the players find out if i missed something